chroma-manager/tests/integration/shared_storage_configuration/test_anonymous_access_control.py

```diff
@@ -1,10 +1,10 @@
 from testconfig import config
-from django.utils.unittest import skipIf
+from django.utils.unittest import skip
 
 from tests.integration.core.chroma_integration_testcase import ChromaIntegrationTestCase
 
 
-@skipIf(config.get('simulator'), 'RealRemoteOperations simulator cant fake out ssh')
+@skip("Needs to work with real cluster")
```

Comment:
Should we open a ticket about this describing why it's being skipped and what needs to be done to remove the skip?

Reply:
Yes, I'll open one.

Reply:
I think this one is backwards actually, I'll post follow up to fix.